Skip integration tests when google application credentials are not available

--- a/tests/integration/smoke_test.py
+++ b/tests/integration/smoke_test.py
@@ -1,4 +1,5 @@
 import datetime
+import os
 
 import pytest
 from cryptography import x509
@@ -16,6 +17,8 @@
 else:
     from aiohttp import ClientSession as Session
 
+pytestmark = pytest.mark.skipif("GOOGLE_APPLICATION_CREDENTIALS" not in os.environ, reason="No credentials")
+
 
 @pytest.mark.asyncio
 async def test_token_is_created(creds: str) -> None:
